actors/cache/impl/zcache.go

```diff
@@ -131,14 +133,17 @@ func (m *ZCache) GetRobustAddress(address address.Address) (string, error) {
 		return "", err
 	}
 
+	ctx := context.Background()
+
 	if isRobustAddress {
-		// Already a robust address
+		if f4Address := m.tryToGetF4Address(address); f4Address != "" {
```

Comment:
Maybe a comment here to say that we are trying to get f4 address if the robust address received is f2 in case of EVM actors would be nice